test/models/employee_test.rb

```diff
@@ -0,0 +1,4 @@
+require "test_helper"
+
+class EmployeeTest < ActiveSupport::TestCase
+end
```

Comment:
I think `EmployeeTest` isn't the best name for a model and folder, what do you think?

Reply:
those are a autogenerate files when run rails generate model, are they wrong?